test/test_tensor_creation_ops.py

```diff
@@ -2952,6 +2948,9 @@ def test_linspace(self, device, dtype):
         # steps = 0
         self.assertEqual(torch.linspace(0, 1, 0, device=device, dtype=dtype).numel(), 0, atol=0, rtol=0)
 
+        # steps not provided
+        self.assertRaises(TypeError, lambda: torch.linspace(0, 1, device=device, dtype=dtype))
```

Comment:
Make this an ErrorInput to the linspace OpInfo

Reply:
resolved offline